src/accounts/AccountPageStyles.scss

```diff
@@ -1,3 +1,24 @@
 .account-settings--header {
   margin-bottom: 16px;
 }
+
+.account-settings--account-name {
+  margin-bottom: 8px;
+}
+
+.account-settings--name-input {
+  width: 250px !important;
```

Comment:
needed to override clockface input style

Reply:
👍

Reply:
Is this absolutely necessary, or can it be overridden with a more specific selector? Maybe something like:

```css
h4.account-settings--header div.account-settings--name-input {
  width: 250px;
```
??

https://css-tricks.com/specifics-on-css-specificity/#aa-calculating-css-specificity-value

Reply:
no, I don't believe thats possible. There is an inline style of `width: 100%` being applied to the element by clockface, we need !important to override it